test/e2e/discovery/headless_services.go

```diff
@@ -13,26 +13,29 @@ import (
 var _ = Describe("[discovery] Test Headless Service Discovery Across Clusters", func() {
 	f := lhframework.NewFramework("discovery")
 
-	if !framework.TestContext.GlobalnetEnabled {
-
-		When("a pod tries to resolve a headless service in a remote cluster", func() {
-			It("should resolve the backing pod IPs from the remote cluster", func() {
+	When("a pod tries to resolve a headless service in a remote cluster", func() {
+		It("should resolve the backing pod IPs from the remote cluster", func() {
+			if !framework.TestContext.GlobalnetEnabled {
```

Comment:
```suggestion
			if !framework.TestContext.GlobalnetEnabled {
				framework.Skipf("Globalnet is not enabled, skipping the test...")
```

Reply:
> framework.Skipf("Globalnet is not enabled, skipping the test...")

But it's not skipping the test, right? It's running it. Copy/paste fragment leftover?

Reply:
I'm not sure I understand the suggestion. Why are we skipping it on both sides of the conditional?

Reply:
Lol,... nevermind I guess my mind was somewhere else, thanks daniel ':D